examples/next/getting-started/src/pages/sitemap.xml.tsx

```diff
@@ -0,0 +1,23 @@
+import "faust.config";
```

Comment:
We'll want to remove this file from the example and leave it up to the user to implement.